common/djangoapps/student/management/commands/change_enrollment.py

```diff
@@ -59,36 +76,70 @@ class Command(BaseCommand):
     )
 
     def handle(self, *args, **options):
+        error_users = []
+        success_users = []
+
         if not options['course_id']:
-            raise CommandError("You must specify a course id for this command")
+            logger.error('You must specify a course id for this command')
```

Comment:
Logging isn't really necessary for these, right? As in, there's no need for this message to go to Splunk. Logging the other changes is good, though.